src/com/activeandroid/DatabaseHelper.java

```diff
@@ -213,7 +213,7 @@ private void executeSqlScript(SQLiteDatabase db, String file) {
 
 		    }
 
-		} catch (IOException e) {
+		} catch (Exception e) {
```

Comment:
Why?

Reply:
> Why?

I wanted to solve  this err `duplicate column name` , you can see issues #542